weave-js/src/components/PagePanelComponents/Home/Browse3/pages/wfReactInterface/traceServerClient.ts

```diff
@@ -94,7 +109,10 @@ export class TraceServerClient extends DirectTraceServerClient {
 
   public feedbackCreate(req: FeedbackCreateReq): Promise<FeedbackCreateRes> {
     const res = super.feedbackCreate(req).then(createRes => {
-      const listeners = this.onFeedbackListeners[req.weave_ref] ?? [];
+      const feedbackTypeResolved =
+        req.feedback_type ?? this.FEEDBACK_TYPE_DEFAULT;
```

Comment:
i think feedback type is always required?

Reply:
Yeah it is, unfortunately because it always exists this now requires changes to the existing feedback listening stuff, working through that now